.githooks/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Colors
+# (http://wiki.bash-hackers.org/snipplets/add_color_to_your_scripts)
+ESC_SEQ="\x1b["
+COL_RESET=$ESC_SEQ"39;49;00m"
+COL_RED=$ESC_SEQ"31;01m"
+COL_YELLOW=$ESC_SEQ"33;01m"
+
+## Checks if a given command exists
+command_exists () {
+  if ! hash "$1" >/dev/null; then
```

Comment:
You might want to redirect error output from the hash command like in [this example](http://stackoverflow.com/a/6227503/1196369)